es6/jsxcad-ui-threejs.js

```diff
@@ -1681,7 +1689,7 @@ const staticDisplay = async (
     canvas,
     geometry,
     withAxes = false,
-    withGrid = true,
+    withGrid = false,
```

Comment:
Is the grid gone by default now?

Reply:
Yeah, I wanted to see how it looks, and if it helps reduce noise in the diffs.

But an explicitly provided value should be honored.

Reply:
I think that makes sense. On the static views I rarely use the grid, and if I wanted to use it I would probably want the top down view